src/shared/avl-consumer/utils.ts

```diff
@@ -1,16 +1,42 @@
+import { z } from "zod";
 import { SubscriptionIdNotFoundError } from "../avl/utils";
-import { queryDynamo } from "../dynamo";
+import { getDynamoItem, queryDynamo, scanDynamo } from "../dynamo";
 import {
     AvlConsumerSubscription,
     AvlSubscriptionStatus,
     avlConsumerSubscriptionSchema,
     avlConsumerSubscriptionsSchema,
 } from "../schema";
+import { createStringLengthValidation } from "../validation";
 
-export const getAvlConsumerSubscriptions = async (tableName: string, status: AvlSubscriptionStatus) => {
-    const subscriptions = await queryDynamo<AvlConsumerSubscription>({
+export const subscriptionTriggerMessageSchema = z.object({
+    subscriptionPK: z.string(),
+    SK: z.string(),
+    frequencyInSeconds: z.union([z.literal(10), z.literal(15), z.literal(20), z.literal(30)]),
+    queueUrl: z.string(),
```

Comment:
We could always enforce a url type here 

```suggestion
    queueUrl: z.string().url(),
```